app/src/main/java/co/touchlab/kampkit/android/metaweather/States.kt

```diff
@@ -0,0 +1,6 @@
+package co.touchlab.kampkit.android.metaweather
+
+enum class States {
+    IN_PROGRESS,
+    FINISHED,
+}
```

Comment:
This can be refactored away from the ViewState as a single boolean `isInProgress: Boolean`